### PR TITLE
added custom pagination

### PR DESCRIPTION
**Pull request by kagz**
the app wont run and wont complain of anything ... despite the  `eslint` warning on the props passed..

### CODE REVIEW

File: src/components/index.jsx [hunk 1]
```diff
@@ -121,9 +147,11 @@ function Home() {
 							}
 							{/* card smaple */}
 						</Row>
-						<StyledPagination>
-							<CustomPagination pageCount={pageCount} handlePageChange={handlePageChange} />
-						</StyledPagination>
+						<LoadMore>
+							<Button disabled={isLoading || hasMore} onClick={loadPage}>
+				Load More
```
Comment (mpalmr) [inline src/components/index.jsx]: Indentation

File: src/components/index.jsx [hunk 2]
```diff
@@ -100,7 +126,7 @@ function Home() {
 														<JobLocation>
 															<li>
 																<Badge pill>
-																	<TimeIcon />	{job.createdAt}
+																	<TimeIcon />	{job.createdAt?.toLocaleString()}
```
Comment (mpalmr) [inline src/components/index.jsx]: Why do you need the `?` here? Isn't it always there or no?

File: src/components/index.jsx [hunk 3]
```diff
@@ -28,19 +29,38 @@ const MainCard = styled.div`
 	box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.5);
 	margin: 7px;
 `;
+const LoadMore = styled.div`
+text-align:center;
```
Comment (mpalmr) [inline src/components/index.jsx]: Indentation

File: src/components/index.jsx [hunk 4]
```diff
@@ -28,19 +29,38 @@ const MainCard = styled.div`
...
+				how_to_apply: '<p>Click this link to apply: <a href="https://tubescience.com/were-hiring/?gh_jid=5039665002">https://tubescience.com/were-hiring/?gh_jid=5039665002</a></p>\n',
+				company_logo: 'https://jobs.github.com/rails/active_storage/blobs/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbStXIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--b8a551a3c84317266c1f3dfe846ded212a933e24/TS%20Linkedin%20Logo.jpeg',
+			},
+		],
+	);
```
Comment (mpalmr) [inline src/components/index.jsx]: Why do we need this object in here?

**Review by mpalmr — CHANGES_REQUESTED** [review]

File: src/components/index.jsx (hunk 2 above)
Comment (kagz) [reply src/components/index.jsx]: without it its complaining when date is null

File: src/components/index.jsx (hunk 4 above)
Comment (kagz) [reply src/components/index.jsx]: the app is not running wthen state is `null` at first...so i added that as the default

File: src/components/index.jsx (hunk 3 above)
Comment (kagz) [reply src/components/index.jsx]: thanks i sorted

File: src/components/index.jsx [hunk 5]
```diff
@@ -64,38 +104,21 @@ function Home() {
...
-
-								{/* <Pagination.Item active>{12}</Pagination.Item> */}
-								<Pagination.Item>{5}</Pagination.Item>
-								{/* <Pagination.Ellipsis /> */}
-								{/* <Pagination.Item>{20}</Pagination.Item> */}
-								{/* <Pagination.Next /> */}
-								{/* <Pagination.Last /> */}
-							</Pagination>
-						</StyledPagination>
+						<LoadMore disabled={isLoading || hasMore} onClick={loadPage}>
```
Comment (mpalmr) [inline src/components/index.jsx]: I'd call this component `LoadMoreButton`

File: src/components/index.jsx [hunk 6]
```diff
@@ -4,36 +4,66 @@ import React, { useState, useEffect } from 'react';
...
+const JOBS_PER_PAGE = 50;
+
 const MainCard = styled.div`
 	padding: 20px;
 	border-radius: 5px;
 	background: #100e17;
 	box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.5);
 	margin: 7px;
 `;
+const LoadMore = styled(Button)`
```
Comment (mpalmr) [inline src/components/index.jsx]: Newline

File: src/styles/index.scss [hunk 7]
```diff
@@ -1,28 +1,44 @@
...
 }
-.page-link:hover {
-	z-index: 2;
-	color: #cdd6e0;
-	text-decoration: none;
-	background-color: #343a40;
-	border: 3px solid #2ecc71;
+
+.active{
+border: 2px solid #000;
```
Comment (mpalmr) [inline src/styles/index.scss]: indentation

File: src/styles/index.scss [hunk 8]
```diff
@@ -1,28 +1,44 @@
...
+
 }
-.page-link:hover {
-	z-index: 2;
-	color: #cdd6e0;
-	text-decoration: none;
-	background-color: #343a40;
-	border: 3px solid #2ecc71;
+
+.active{
```
Comment (mpalmr) [inline src/styles/index.scss]: Spacing

File: src/styles/index.scss [hunk 9]
```diff
@@ -1,28 +1,44 @@
...
-	padding: 0;
-	position: relative;
-	justify-content: center;
-	align-items: center;
-	border: 3px solid #2ecc71;
+	list-style: none;
+}
+
+.page{
+padding: 10px;
```
Comment (mpalmr) [inline src/styles/index.scss]: Indentaiton

File: src/styles/index.scss [hunk 10]
```diff
@@ -1,28 +1,44 @@
 @import "bootstrap";
-.page-link,
-.page-item:first-child .page-link,
-.page-item:last-child .page-link {
-	width: 40px;
+.container{
```
Comment (mpalmr) [inline src/styles/index.scss]: Spacing

**Review by mpalmr — CHANGES_REQUESTED** [review]

File: src/components/index.jsx (hunk 4 above)
Comment (mpalmr) [reply src/components/index.jsx]: So show a loading state when it's `null`. If you don't wan ta loading state (which I think you do) make the default value `[]`.

**Pull request merged**